Projects/App/Sources/MyBox/MyBoxViewController.swift

```diff
@@ -105,9 +108,11 @@ final class MyBoxViewController: BaseViewController<MyBoxViewModelProtocol> {
     
     @objc private func swipeAction(_ gesture: UISwipeGestureRecognizer) {
         if gesture.direction == .left {
+            categoryTapView.tapRightButton()
             myBoxView.collectionView.reloadData()
             return
         }
+        categoryTapView.tapLeftButton()
         myBoxView.collectionView.reloadData()
```

Comment:
```suggestion
        if gesture.direction == .left {
            categoryTapView.tapRightButton()
        } else {
            categoryTapView.tapLeftButton()        
        }
        myBoxView.collectionView.reloadData()
```

위와같이 중복되는 코드를 삭제할 수 있을 것 같습니다

Reply:
반영하였습니다!